linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_su/rule.yml

```diff
@@ -1,7 +1,13 @@
-{{%- if product in ["fedora", "ol7", "ol8", "ol9", "rhcos4", "rhel7", "rhel8", "rhel9", "sle12", "sle15", "ubuntu2004", "ubuntu2204"] %}}
+{{%- if product in ["fedora", "ol7", "ol8", "ol9", "rhcos4", "rhel7", "rhel8", "rhel9", "sle12", "sle15", "ubuntu2204"] %}}
   {{%- set perm_x="-F perm=x " %}}
 {{%- endif %}}
 
+{{%- if product in ["ubuntu2004"] %}}
+  {{%- set su_path="/bin/su" %}}
+{{%- else %}}
+  {{%- set su_path="/usr/bin/su" %}}
+{{% endif %}}
+
```

Comment:
I don't think this is necessary, `/usr/bin/su` should symlink to `/bin/su`. Despite it showing on DISA that the path should be `/bin/su`, this should still be valid.

Reply:
yes, that's not needed